qutebrowser/config/configdata.py

```diff
@@ -1196,6 +1200,18 @@ def data(readonly=False):
              "Background color for webpages if unset (or empty to use the "
              "theme's color)"),
 
+            ('keyhint.fg',
+             SettingValue(typ.QssColor(), '#FFFFFF'),
+             "Text color for the keyhint widget."),
+
+            ('keyhint.fg.suffix',
+             SettingValue(typ.QssColor(), '#FFFF00'),
```

Comment:
I think this should be `CssColor` as the color is used inside HTML and not a Qt stylesheet.